docs/source/quickstarts/gke.md

```diff
@@ -0,0 +1,91 @@
+# Deploying ScyllaDB on GKE
+
+This is a quickstart guide to help you set up a basic GKE cluster quickly with local NVMes and solid performance.
+
+This is by no means a complete guide, and you should always consult your provider's documentation.
+
+
+## Creating a GKE cluster
+
+First, we need to create a kubelet config to configure [static CPU policy](../installation/kubernetes/generic.md#static-cpu-policy)
+:::{code} bash
+:number-lines:
+
+cat > systemconfig.yaml <<EOF
+kubeletConfig:
+  cpuManagerPolicy: static
+EOF
+:::
+
+
+Then we'll create a GKE cluster with the following:
+
+:::{code} bash
+
+gcloud container \
+clusters create 'my-k8s-cluster' \
+--zone='us-central1' \
+--machine-type "n1-standard-8" \
+--num-nodes "2" \
+--disk-type "pd-ssd" --disk-size "20" \
+--image-type "UBUNTU_CONTAINERD" \
+--enable-stackdriver-kubernetes
+:::
+
+and then we'll a dedicated pool with NVMes for ScyllaDB
```

Comment:
```suggestion
and then we'll create a dedicated pool with NVMes for ScyllaDB
```

or `and then a dedicated pool with NVMes for ScyllaDB`

Reply:
fixed